ui/src/app/utils/access-utils.tsx

```diff
@@ -171,7 +171,7 @@ export const getAccessModuleConfig = (
     accessModules,
   } = serverConfigStore.get().config;
   const apiConfig = accessModules.find((m) => m.name === moduleName);
-  return switchCase(
+  return switchCase<AccessModule, any>(
```

Comment:
This any means any of the keys below. Not sure it's worth attempting to type this properly.